Makefile

```diff
@@ -59,9 +59,28 @@ prow-jobs:
 	oc process -f cluster/ci/jobs/commenter.yaml | oc apply -f -
 .PHONY: prow-jobs
 
-mungegithub:
+mungegithub: submit-queue-secrets origin-submit-queue installer-submit-queue logging-submit-queue
 .PHONY: mungegithub
 
+submit-queue-secrets:
+	# HMAC_TOKEN is used for encrypting Github webhook payloads.
+	oc create secret generic sq-hmac-token --from-literal=token=${HMAC_TOKEN} -o yaml --dry-run | oc apply -f -
```

Comment:
we should really use the same secrets here in the future

Reply:
Same with the rest of the queues or same with the prow hmac token?

Reply:
queues already all share them -- but the HMAC should be shared with Prow, right?

Reply:
Not necessarily. Tide won't need it at all.